pkg/db/db_local/service.go

```diff
@@ -22,9 +22,9 @@ func GetState() (*RunningDBInstanceInfo, error) {
 	if err != nil {
 		return nil, err
 	}
+	log.Printf("[TRACE] GetState - loadRunningInstanceInfo returned %#v\n", info)
```

Comment:
This logs the database password, not sure if this is an issue in TRACE log level.

Reply:
Removed.